test/intl402/Temporal/TimeZone/from/timezone-case-insensitive.js

```diff
@@ -3,10 +3,615 @@
+  'Asia/Chungking',
+  'Asia/Dacca',
+  'Asia/Harbin',
+  'Asia/Istanbul',
+  'Asia/Kashgar',
+  'Asia/Katmandu',
+  'Asia/Kuala_Lumpur',
+  'Asia/Kuwait',
+  'Asia/Macao',
+  'Asia/Muscat',
+  'Asia/Phnom_Penh',
+  'Asia/Rangoon',
+  'Asia/Saigon',
+  'Asia/Tel_Aviv',
+  'Asia/Thimbu',
+  'Asia/Ujung_Pandang',
+  'Asia/Ulan_Bator',
+  'Asia/Vientiane',
+  'Atlantic/Faeroe',
+  'Atlantic/Jan_Mayen',
+  'Atlantic/Reykjavik',
+  'Atlantic/St_Helena',
+  'Australia/ACT',
+  'Australia/Canberra',
+  'Australia/Currie',
+  'Australia/LHI',
+  'Australia/NSW',
+  'Australia/North',
+  'Australia/Queensland',
+  'Australia/South',
+  'Australia/Tasmania',
+  'Australia/Victoria',
+  'Australia/West',
+  'Australia/Yancowinna',
+  'Brazil/Acre',
+  'Brazil/DeNoronha',
+  'Brazil/East',
+  'Brazil/West',
+  'Canada/Atlantic',
+  'Canada/Central',
+  'Canada/Eastern',
+  'Canada/Mountain',
+  'Canada/Newfoundland',
+  'Canada/Pacific',
+  'Canada/Saskatchewan',
+  'Canada/Yukon',
+  'Chile/Continental',
+  'Chile/EasterIsland',
+  'Cuba',
+  'Egypt',
+  'Eire',
+  'Etc/GMT+0',
+  'Etc/GMT-0',
+  'Etc/GMT0',
+  'Etc/Greenwich',
+  'Etc/UCT',
+  'Etc/Universal',
+  'Etc/Zulu',
+  'Europe/Amsterdam',
+  'Europe/Belfast',
+  'Europe/Bratislava',
+  'Europe/Busingen',
+  'Europe/Copenhagen',
+  'Europe/Guernsey',
+  'Europe/Isle_of_Man',
+  'Europe/Jersey',
+  'Europe/Kiev',
+  'Europe/Ljubljana',
+  'Europe/Luxembourg',
+  'Europe/Mariehamn',
+  'Europe/Monaco',
+  'Europe/Nicosia',
+  'Europe/Oslo',
+  'Europe/Podgorica',
+  'Europe/San_Marino',
+  'Europe/Sarajevo',
+  'Europe/Skopje',
+  'Europe/Stockholm',
+  'Europe/Tiraspol',
+  'Europe/Uzhgorod',
+  'Europe/Vaduz',
+  'Europe/Vatican',
+  'Europe/Zagreb',
+  'Europe/Zaporozhye',
+  'GB',
+  'GB-Eire',
+  'GMT',
+  'GMT+0',
+  'GMT-0',
+  'GMT0',
+  'Greenwich',
+  'Hongkong',
+  'Iceland',
+  'Indian/Antananarivo',
+  'Indian/Christmas',
+  'Indian/Cocos',
+  'Indian/Comoro',
+  'Indian/Kerguelen',
+  'Indian/Mahe',
+  'Indian/Mayotte',
+  'Indian/Reunion',
+  'Iran',
+  'Israel',
+  'Jamaica',
+  'Japan',
+  'Kwajalein',
+  'Libya',
+  'Mexico/BajaNorte',
+  'Mexico/BajaSur',
+  'Mexico/General',
+  'NZ',
+  'NZ-CHAT',
+  'Navajo',
+  'PRC',
+  'Pacific/Chuuk',
+  'Pacific/Enderbury',
+  'Pacific/Funafuti',
+  'Pacific/Johnston',
+  'Pacific/Majuro',
+  'Pacific/Midway',
+  'Pacific/Pohnpei',
+  'Pacific/Ponape',
+  'Pacific/Saipan',
+  'Pacific/Samoa',
+  'Pacific/Truk',
+  'Pacific/Wake',
+  'Pacific/Wallis',
+  'Pacific/Yap',
+  'Poland',
+  'Portugal',
+  'ROC',
+  'ROK',
+  'Singapore',
+  'Turkey',
+  'UCT',
+  'US/Alaska',
+  'US/Aleutian',
+  'US/Arizona',
+  'US/Central',
+  'US/East-Indiana',
+  'US/Eastern',
+  'US/Hawaii',
+  'US/Indiana-Starke',
+  'US/Michigan',
+  'US/Mountain',
+  'US/Pacific',
+  'US/Pacific-New',
+  'US/Samoa',
+  'UTC',
+  'Universal',
+  'W-SU',
+  'Zulu'
+];
+
+const ids = [...new Set([...timeZoneIdentifiers, ...Intl.supportedValuesOf('timeZone')])];
```

Comment:
Not sure I understand the purpose of this — it's a bit puzzling to me why we need to use both the hardcoded list of identifiers and the `supportedValuesOf` list. Maybe an explanatory comment would be helpful?

Reply:
OK will add one. `supportedValuesOf` was included to handle the case of new IDs being added before these tests are updated to include those new IDs. The reason why `supportedValuesOf` is insufficient is because it only returns canonical IDs. If you also want to test non-canonical IDs ("non-primary time zone identifiers") in official spec terminology) then you need to hard-code a list in the test.